go/internal/openapi/api_environment.go

```diff
@@ -1,9 +1,9 @@
+ * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
```

Comment:
Why are all these comments changing? I have no idea of the practical use of them in the first place, but is "No description provided" what we want now?

Reply:
Fair enough.